apps/beets-frontend-v3/lib/modules/lst/components/LstUnstakeSummary.tsx

```diff
@@ -25,21 +19,12 @@ export function LstUnstakeSummary() {
       <Card variant="modalSubSection">
         <LstTokenRow
           chain={chain}
-          isLoading={isLoading}
+          isLoading={false}
```

Comment:
Tiny improvement: default prop to false so that it is not required.

Reply:
done!